autodoc.pl

```diff
@@ -1369,11 +1369,13 @@ ($$$)
             my @base_args = $items[0]->{args}->@*;
             my $base_thread_context = $items[0]->{flags} =~ /T/;
             for (my $i = 1; $i < @items; $i++) {
-                no warnings 'experimental::smartmatch';
                 my $item = $items[$i];
+                my $identical_args_count;
+                $item->{args}[$_] eq $base_args[$_] and $identical_args_count++
+                  for 0 .. $#base_args;
```

Comment:
FYI: This is fine as-is, but don't forget that `List::Util::all` exists. A neater way to write this could be

```perl
use List::Util 'all';
my $args_are_equal = @baseargs == $item->{args}->@* and
    all { $baseargs[$_] eq $item->{args}{$_} } keys @baseargs;
```